artifactor/plugins/reporter.py

```diff
@@ -131,8 +131,6 @@ def process_data(self, artifacts, log_dir, version, name_filter=None):
             if not test.get('old', False):
                 current_counts[overall_status] += 1
             color = colors[overall_status]
-            # Set the overall status and then process duration
```

Comment:
This assumed that it was mutating the dict, but with the composite plugin, things don't happen like that any more, so it has to be passed back out by ```finish_test``` hook.